static/styles/master.css

```diff
@@ -1,3 +1,21 @@
 body {
     font-family: 'Maven Pro', sans-serif;
+}
+
+
+label{
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 24px;
+}
+
+input{
+    border: 3px solid red;
+}
+
+.border-class
+{
```

Comment:
Make sure to follow the style convention with space between class and bracket and no extra line breaks like before the closing bracket.

Reply:
I have corrected all errors from the pr, most of them were because I pushed to the remote branch an outdated version of the component.

Reply:
All changes have been applied.